waypoint_x: [0.3, 0.7, 0.7, 0.5, 0.3] # The x coordinates for a pentagonal path
waypoint_y: [0.2, 0.2, 0.5, 0.7, 0.5] # The y coordinates for a pentagonal path
frequency: 110 # Rate to send commands to the turtlebot
frac_val: .55 # velocity setting, 1 is max vel.
kp: 0.25 # proportional gain
